.pre-commit-config.yaml

```diff
@@ -1,3 +1,5 @@
+ci:
+    skip: [clang-format]
```

Comment:
What does this do?

Reply:
Prevents pre-commit from doing its thing like here: https://github.com/eic/EICrecon/pull/848/commits/0914efe0268a6a0f04bd960c78f642a6fe49eb09. It makes sure that clang-format is only going in after review by a human and won't automatically mess up someone's carefully crafted alignment.